classes/helper.php

```diff
@@ -124,6 +126,20 @@ public static function http_status_display(int $status): string {
         return \html_writer::tag('span', $status, ['class' => $spanclass]);
     }
 
+    /**
+     * Returns status as a badge.
+     *
+     * @param int $status
+     * @return string
+     */
+    public static function status_display(int $status): string {
+        if ($status < 100) {
```

Comment:
I think this logic will need either the full record as input so it can look at the script type

"Exit codes are a number between 0 and 255, which is returned by any Unix command when it returns control to its parent process."

https://www.shellscript.sh/exitcodes.html

Reply:
The spanner is /admin/cron.php, which produces a HTTP response code, but is flagged as a CLI script internally. Haven't yet figured out what to do about it.

Reply:
If status doesn't make sense then I'd just leave it as null. In theory it has a status but we can't get at it easily

Reply:
Sorry I misunderstood that and was talking about a task not the whole process. In any case it's such a small edge case I don't want to worry about it now

Reply:
I left a TODO note for future reference.